package.json

```diff
@@ -105,6 +105,7 @@
     "debug": "^4.3.1",
     "dgram": "^1.0.1",
     "err-code": "^3.0.1",
+    "prom-client": "^14.1.0",
```

Comment:
I don't think discv5 should depend on prom-client directly. Instead just expect an specific interface like gossipsub